.github/workflows/test.yml

```diff
@@ -10,23 +10,25 @@ concurrency:
 
 jobs:
   tests:
-    name: Python ${{ matrix.python-version }}, Database ${{ matrix.database-type }}
+    name: Python ${{ matrix.python-version[0] }}, Database ${{ matrix.database-type }}
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: false
       matrix:
         python-version:
-          - "3.7"
-          - "3.8"
-          - "3.9"
-          - "3.10"
-          - "pypy-3.7"
-          - "pypy-3.8"
-          - "pypy-3.9"
+          - ["3.7", "py37"]
+          - ["3.8", "py38"]
+          - ["3.9", "py39"]
+          - ["3.10", "py310"]
+          - ["pypy-3.7", "pypy37"]
+          - ["pypy-3.8", "pypy38"]
+          - ["pypy-3.9", "pypy39"]
         database-type:
           - "sqlite"
           - "postgres"
+        run-coverage:
+          - "${{ (github.ref == 'refs/heads/master' || contains(github.event.commits[0].message, 'coverage')) && 'yes' || 'no' }}"
```

Comment:
How about this?

* `master` takes [3:10s](https://github.com/FactoryBoy/factory_boy/actions/runs/2578141014), [3:45s](https://github.com/FactoryBoy/factory_boy/actions/runs/2575372482), [3:50s](https://github.com/FactoryBoy/factory_boy/actions/runs/2578163487) to run without coverage
* this branch without coverage takes [4:13s](https://github.com/FactoryBoy/factory_boy/actions/runs/2579118949) but I suspect it's just github action runners just being a little slower just cause. and with coverage [5:26s](https://github.com/kingbuzzman/factory_boy/actions/runs/2579194000)

The idea is any branch you're working on doesn't run coverage unless it's master/main, and if you want coverage in your local branch to see whats going on.. you can add the string "`coverage`" to your commit message

Reply:
This has been removed